src/issuingPurchase/issuingPurchase.php

```diff
@@ -67,28 +70,31 @@ class IssuingPurchase extends Resource
         - merchantCurrencyCode [string]: merchant currency code. ex: "USD"
         - merchantCurrencySymbol [string]: merchant currency symbol. ex: "$"
         - merchantCategoryCode [string]: merchant category code. ex: "fastFoodRestaurants"
+        - merchantCategoryType [string]: merchant category type. ex: "food"
         - merchantCountryCode [string]: merchant country code. ex: "USA"
         - acquirerId [string]: acquirer ID. ex: "5656565656565656"
         - merchantId [string]: merchant ID. ex: "5656565656565656"
         - merchantName [string]: merchant name. ex: "Google Cloud Platform"
-        - metadata [Dictionary object]: object used to store additional information about the IssuingPurchase object. ex: [authorizationId => 'OjZAqj']
         - merchantFee [integer]: fee charged by the merchant to cover specific costs, such as ATM withdrawal logistics, etc. ex: 200 (= R$ 2.00)
         - walletId [string]: virtual wallet ID. ex: "5656565656565656"
         - methodCode [string]: method code. Options: "chip", "token", "server", "manual" or "contactless"
         - score [float]: internal score calculated for the authenticity of the purchase. null in case of insufficient data. ex: 7.6
         - endToEndId [string]: Unique id used to identify the transaction through all of its life cycle, even before the purchase is denied or accepted and gets its usual id. ex: "679cd385-642b-49d0-96b7-89491e1249a5"
         - tags [array of string]: array of strings for tagging returned by the sub-issuer during the authorization. ex: ["travel", "food"]
-        - zipCode [string]: zip code of the merchant location. ex: "02101234"
-
+        
     ## Attributes (IssuingPurchase only):
         - issuingTransactionIds [string]: ledger transaction ids linked to this Purchase
         - status [string]: current IssuingCard status. Options: "approved", "canceled", "denied", "confirmed", "voided"
-        - updated [DateTime]: latest update datetime for the IssuingPurchase.
+        - description [string]:  IssuingPurchase description. ex: "Office Supplies"
+        - metadata [dictionary object]: object used to store additional information about the IssuingPurchase object. ex: [authorizationId => "OjZAqj"]
```

Comment:
dictionary object used to store additional information about the Transfer object. ex: [authorizationId => "OjZAqj"]